sympy/concrete/summations.py

```diff
@@ -989,22 +991,21 @@ def eval_sum_symbolic(f, limits):
         r = gosper_sum(f, (i, a, b))
 
         # to find Sum at points where denominator approaches to zero
-        from sympy import together, fraction, Mul, Add, ordered
-        if isinstance(r, (Mul,Add)) and not(any(x in r.atoms(Symbol) for x in limits)):
-            var = ordered(r.atoms(Symbol))
-            _, denom = fraction(together(r))
-            if any(x in denom.atoms(Symbol) for x in var):
+        if isinstance(r, (Mul,Add)):
+            from sympy import ordered
+            var = r.free_symbols
+            var = [x for x in var if x not in limits]
```

Comment:
I would prefer to see this be `var = r.free_symbols & Tuple(*limits).free_symbols` or perhaps `var = r.free_symbols & Tuple(*limits[1:]).free_symbols` since then there is no ambiguity.

Reply:
@smichr actually `var` should be free from any `Symbol` inside `limits`. I think the above one would give `Symbols` common in `r` and `linits`.

Reply:
How about : ` var = [x for x in r.free_symbols if x not in limits]`?